lightly_studio_view/src/lib/components/PlotPanel/usePlotData.ts

```diff
@@ -0,0 +1,91 @@
+import { writable, type Writable } from 'svelte/store';
+import { EmbeddingView, type Point } from 'embedding-atlas/svelte';
+import type { ComponentProps } from 'svelte';
+import type { ArrowData } from './useArrowData';
+import { isPointInPolygon } from '$lib/hooks/useEmbeddingSelection/isPointInPolygon';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const plotColumns = ['x', 'y', 'category'] as const;
+type PlotColumn = (typeof plotColumns)[number];
```

Comment:
I am not sure I get the need for `plotColumns` - can't PlotColumn just be defined as a union type?

```suggestion
type PlotColumn = 'x' | 'y' | 'category';
```